homeassistant/components/prusalink/__init__.py

```diff
@@ -201,7 +201,7 @@ async def _fetch_data(self) -> JobInfo:
         return await self.api.get_job()
 
 
-class PrusaLinkEntity(CoordinatorEntity[PrusaLinkUpdateCoordinator]):  # pylint: disable=hass-enforce-coordinator-module
+class PrusaLinkEntity(CoordinatorEntity[PrusaLinkUpdateCoordinator]):
```

Comment:
There's no coordinator module. Why should we remove the disable?

Reply:
Because the pylint plugin only triggers on the coordinator class itself, not on the entity classes